frontend/lib/norent/letter-content.tsx

```diff
@@ -85,17 +87,26 @@ const LetterHeading: React.FC<NorentLetterContentProps> = (props) => (
   </dl>
 );
 
-const TenantProtections: React.FC<{}> = () => (
-  <>
-    <p>
-      Tenants impacted by the COVID-19 crisis are protected from eviction for
-      nonpayment per emergency declaration(s) from:
-    </p>
-    <ul>
-      <li>US Congress, CARES Act (Title IV, Sec. 4024), March 27, 2020</li>
-    </ul>
-  </>
-);
+const TenantProtections: React.FC<NorentLetterContentProps> = (props) => {
+  const state = props.state as USStateChoice;
```

Comment:
hmm so one function I added to help w/ this situation is the `assertIsUSState()` function that's in `national-metadata.tsx`... it will both ensure that the state is a valid choice and return a type of `USStateChoice` so you don't have to do the typecast here.  I think it'd be best to use it in the spirit of [crash early, crash often](https://medium.com/@mattklein123/crash-early-and-crash-often-for-more-reliable-software-597738dd21c5) but if we're not using it then I should probably remove that function.

Reply:
Oh good to know! I saw that function but it didn't occur to me that it actually would return the state in the correct type. I'll keep this in mind for when we revisit these things